### DISCUSSION TITLE
Use Azure AD for LDAP login

### DISCUSSION THREAD

**Discussion by williwehrlos** [goauthentik/authentik#7210] [Q&A]
Hello,

is it possible to use the Azure AD source with the ldap outpost somehow? So that you can authenticate a user via ldap with his Microsoft credentials.

Regards

**Comment by rqi14** [comment]
You can configure LDAPS directly with Azure AD https://learn.microsoft.com/en-us/entra/identity/domain-services/tutorial-configure-ldaps
